app/scripts/modules/core/chaosMonkey/chaosMonkeyExceptions.component.spec.ts

```diff
@@ -33,13 +33,13 @@ describe('Controller: ChaosMonkeyExceptions', () => {
 
     it('gets all accounts, then adds wildcard and regions per account to vm', () => {
       let accounts = [ {name: 'prod'}, {name: 'test'} ];
-      let details = {
+      let details: any = {
         prod: { name: 'prod', regions: [ {name: 'us-east-1'}, {name: 'us-west-1'}] },
         test: { name: 'test', regions: [ {name: 'us-west-2'}, {name: 'eu-west-1'}] }
       };
 
       spyOn(accountService, 'listAccounts').and.returnValue($q.when(accounts));
-      spyOn(accountService, 'getAccountDetails').and.callFake((accountName) => {
+      spyOn(accountService, 'getAccountDetails').and.callFake((accountName: string): ng.IPromise<any> => {
```

Comment:
Linter was unhappy. I threw so many different `any`s on this block of code to appease the linter and this was the least impactful change I could find.

It might be that the interface for `ng.IPromise` is generic, so it wants something in there for the `T`